src/Cedar/Proto_PPP.c

```diff
@@ -3298,7 +3303,7 @@ bool PPPProcessEAPTlsResponse(PPP_SESSION *p, PPP_EAP *eap_packet, UINT eapTlsSi
 				else
 				{
 					PPP_PACKET *pack;
-					UINT identificator = p->Eap_PacketId - 1; // THIS IS A HACK TO SUPPORT VPN Client Pro on Android!!!

```

Comment:
Does this mean such a hack is no longer needed?

Reply:
This is not a hack actually. Unlike EAP Request where identifier should be increasing, EAP Success should use the same identifier as the previous Response. VPN Client Pro is behaving correctly.

Reply:
I removed this line because `Eap_PacketId` is not auto incremented now. `NextId` is.